.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Bug report
+about: Report an issue with this library
+---
+<!-- Please search existing issues and discussions to avoid creating duplicates. -->
+
+- OpenPGP.js version:
+- Affected platform (Web, Node.js):
```

Comment:
Last suggestion: it might be good to ask for browser+version or Node.js version here